solver/src/main/java/org/chocosolver/solver/Model.java

```diff
@@ -651,6 +657,13 @@ public Settings getSettings() {
         return this.settings;
     }
 
```

Comment:
> Thank you for the trace, it is easier to get an idea on the result.
> 
> ```
> IntVar
>     BitsetIntVarImpl = 73
>         - Nb instantiated: 0
>         - Nb constants: 0
>         - Nb unconstrained: 0
>         - By domain size: {10-100: 16, 101-1000: 9, >1001: 48}
>         - By number of propagators: {2: 58, 7: 14, 8: 1}
>         - By number of views: {0: 73}
> ...
> ```
> 
> I think `Nb constants` is redundant with `FixedIntVarImpl ` (well, that's because I know that those objects represent constant). Otherwise, that quite readable and informative !

Yes, given the current structure I gave to the analysis objects, the nbConstants statistics is never useful. I will remove it for now.